eva_submission/eload_deletion.py

```diff
@@ -0,0 +1,123 @@
+import gzip
+import os
+import shutil
+import tarfile
+from pathlib import Path
+
+from ebi_eva_common_pyutils.config import cfg
+
+from eva_submission.eload_submission import Eload
+from eva_submission.submission_in_ftp import deposit_box
+
+
+class EloadDeletion(Eload):
+    def __init__(self, eload_number):
+        super().__init__(eload_number)
+        self.project_accession = self.eload_cfg.query('brokering', 'ena', 'PROJECT')
+        self.project_dir = os.path.join(cfg['projects_dir'], self.project_accession)
+        self.lts_archive_file = os.path.join(cfg['eloads_lts_dir'], f'{self.eload}.tar')
+
+    def delete_submission(self, ftp_box, submitter, force_delete=False):
+        # check if already present in LTS
+        if os.path.exists(self.lts_archive_file) and not force_delete:
+            raise Exception(
+                f'File already exists in the LTS for the eload {self.eload_num}. LTS file: {self.lts_archive_file}')
+
+        self.upgrade_to_new_version_if_needed()
+        self.archive_eload()
+
+        # delete
+        ftp_dir = deposit_box(ftp_box, submitter)
+        self.delete_ftp_dir(ftp_dir)
```

Comment:
I just realised that we might need to think about this part a bit as there might be cases where these director won't exists. 
Can we make this deletion optional?

Reply:
Before deleting, we do check if the path exists. See [here](https://github.com/EBIvariation/eva-submission/blob/8a8a069b8236017723f8f74d127f148d277b6b69/eva_submission/eload_deletion.py#L112)